.prettierrc.yml

```diff
@@ -7,3 +7,4 @@ singleQuote: true
 trailingComma: 'all'
 bracketSpacing: true
 arrowParens: 'always'
+singleAttributePerLine: true
```

Comment:
New attribute added in Prettier v2.6: https://prettier.io/blog/2022/03/16/2.6.0.html